scripts/apitypings/main.go

```diff
@@ -172,3 +173,47 @@ func FixSerpentStruct(gen *guts.Typescript) {
 		}
 	})
 }
+
+// SerializeTypes serializes the generated TS in a stable order and filters
+// out declarations we do not want to emit (e.g., WildcardValue).
+func SerializeTypes(ts *guts.Typescript) (string, error) {
+	return ts.SerializeInOrder(func(nodes map[string]bindings.Node) []bindings.Node {
+		type pair struct {
+			name string
+			node bindings.Node
+		}
+		list := make([]pair, 0, len(nodes))
+		for _, n := range nodes {
+			var name string
+			switch v := n.(type) {
+			case *bindings.Interface:
+				if v.Name.Ref() == "WildcardValue" {
+					continue
+				}
+				name = v.Name.Ref()
+			case *bindings.Alias:
+				if v.Name.Ref() == "WildcardValue" {
+					continue
+				}
+				name = v.Name.Ref()
+			case *bindings.Enum:
+				name = v.Name.Ref()
+			case *bindings.VariableStatement:
+				if v.Declarations != nil && len(v.Declarations.Declarations) > 0 {
+					name = v.Declarations.Declarations[0].Name.Ref()
+				} else {
+					name = "var:"
+				}
+			default:
+				name = fmt.Sprintf("%T", n)
+			}
+			list = append(list, pair{name: name, node: n})
+		}
+		sort.Slice(list, func(i, j int) bool { return list[i].name < list[j].name })
+		out := make([]bindings.Node, 0, len(list))
+		for _, p := range list {
+			out = append(out, p.node)
+		}
+		return out
+	})
+}
```

Comment:
What does this actually do in the typesgenerated diff?

I think we can remove this, and go back to just `ts.Serialize()` because `WildCard` type was removed.

`guts` can omit some types with either a [comment](https://github.com/coder/gots/blob/main/convert.go#L222-L224) or in the generator

https://github.com/toqueteos/guts/blob/e8d8e5e2eb31376624b9c9cb1bb4eabf20f69aa7/example/simple/main.go#L31-L32